samples/embed/teste.c

```diff
@@ -41,9 +44,26 @@ static void* custom_malloc(size_t bytes)
 	return malloc(bytes);
 }
 
+#ifdef _TESTCASE_
+#ifdef __cplusplus
+extern "C"
+#endif
+int
+test_mono_embed_main (void);
+
 int 
-main(int argc, char* argv[]) {
+test_mono_embed_main() {
+#else
+int main() {
+#endif
+
 	MonoDomain *domain;
+	int argc = 2;
+    char *argv[] = {
+                        (char*)"test-mono-embed.exe",
+                        (char*)"test-embed.exe",
+                        NULL
+                    };
```

Comment:
1. Formatting: should be tabs, not spaces.
2. Missing `#if _TESTCASE_` ?

Reply:
I let fixed the file name and changed the comment about how to run in the beginning of the source code.